File: class05/cpp/AP0501-OlaNomeIdade.cpp
#include <iostream>
#include <string>
int main() {
    std::string nome; int idade;
    std::cout << "Nome: "; std::getline(std::cin, nome);
    std::cout << "Idade: "; std::cin >> idade;
    std::cout << "Olá, " << nome << "! Você tem " << idade << " anos." << std::endl;
    return 0;
}

File: class05/java/AP0501-OlaNomeIdade.java
import java.util.Scanner;
public class Saudacao {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Nome: "); String nome = sc.nextLine();
        System.out.print("Idade: "); int idade = sc.nextInt();
        System.out.println("Olá, " + nome + "! Você tem " + idade + " anos.");
    }
}
